### PR TITLE
Removed view/Dependencies from asmtools URL

### PR DESCRIPTION
**Pull request by IamLRBA**
As per the issue [546](https://github.com/adoptium/TKG/issues/546)
cc
@llxia 
@smlambert

### CODE REVIEW

**Comment by llxia** [comment]
Thanks @IamLRBA . Please remove your unrelated local changes

image

**Review by llxia — CHANGES_REQUESTED** [review]
See https://github.com/adoptium/TKG/pull/643#issuecomment-2506342743

File: scripts/getDependencies.pl [hunk 1]
```diff
@@ -125,12 +125,12 @@
 		sha1 => 'bfcb96281ea3b59d626704f74bc6d625ff51cbce'
 	},
 	asmtools => {
-		url => 'https://ci.adoptium.net/view/Dependencies/job/dependency_pipeline/lastSuccessfulBuild/artifact/asmtools/asmtools-core-7.0.b10-ea.jar',
-		fname => 'asmtools.jar',
-		shaurl => 'https://ci.adoptium.net/view/Dependencies/job/dependency_pipeline/lastSuccessfulBuild/artifact/asmtools/asmtools-core-7.0.b10-ea.jar.sha256sum.txt',
-		shafn => 'asmtools.jar.sha256sum.txt',
-		shaalg => '256'
-	},
+        url => 'https://ci.adoptium.net/job/dependency_pipeline/lastSuccessfulBuild/artifact/asmtools/asmtools-core-7.0.b10-ea.jar',
```
Comment (llxia) [inline scripts/getDependencies.pl:128]: Please fix the indentation. The original code is correct.

**Comment by IamLRBA** [comment]
I have created a new [PR](https://github.com/adoptium/TKG/compare/master...IamLRBA:TKG:master) that should have the corrections addressed. You could look at it.
Thank You!

**Pull request closed without merging**